sharing-k8s-dashboard/README.md

```diff
@@ -0,0 +1,37 @@
+This is an example of using Argo Tunnel to share dashboard your local kubernetes cluster with your collaborators. It is generally not recommended to expose dashboard of production clusters. If you need to,
```

Comment:
Can we physically wrap these lines at ~80 chars like in other files?

Reply:
Sure, it's done now.